scripts/probe_xas_nrtidalv3_phase.py

```
@@ -0,0 +1,239 @@
+        (jnp.array(fs), NR_phase, valid),
+    )
+    P_P = general_planck_taper(jnp.array(fs), 1.15 * f_mrg_ripple, 1.35 * f_mrg_ripple)
+    PN_phase = get_tidal_phase_PN(x_pi_f_Ms, Xa_jnp, l1, l2, PN_coeffs)
+    ripple_phi_tidal_NRblend = NR_phase_clamped * (1 - P_P) + PN_phase * P_P
+
+    # Note: LAL also adds spin-induced 2PN/3PN/3.5PN terms to phaseTidal in the
+    # main loop. Ripple keeps these separate (psi_QM + psi_SS). For an
+    # apples-to-apples we just compare the NR/PN-blend slice (which is what
+    # LAL's SimNRTunedTidesFDTidalPhaseFrequencySeries returns).
+    diff = np.array(ripple_phi_tidal_NRblend) - lal_phi_tidal
+    print(f"(2b) ripple psi_T (NR/PN blend only) - LAL phi_tidal[idx]:")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)